packages/rancher-vsphere-csi/charts/templates/vsphere-csi-controller-deployment.yaml

```diff
@@ -92,6 +106,9 @@ spec:
             - name: healthz
               containerPort: 9808
               protocol: TCP
+            - name: prometheus
```

Comment:
nit: was this a copy paste? If this was from upstream that's okay, but if it is from us I think we should modify this name to `metrics` instead of `prometheus`

Reply:
Change from upstream as well